layers/+source-control/git/packages.el

```diff
@@ -358,7 +358,7 @@ Press [_b_] again to blame further in the history, [_q_] to go up or quit."
         'spacemacs/magit-toggle-whitespace)
       ;; full screen magit-status
       (when git-magit-status-fullscreen
-        (setq magit-display-buffer-function 'spacemacs//fullscreen-magit)))))
+        (setq magit-display-buffer-function 'magit-display-buffer-fullframe-status-v1)))))
```

Comment:
Yes that's possible. Also those documentation sections in the README could also be cleaned up a bit as well to use the layer variable system rather than defining them in `user-init`